cl/corpus_importer/management/commands/harvard_merge.py

```diff
@@ -691,6 +720,10 @@ def merge_opinion_clusters(
         logger.warning(
             msg=f"Cluster source exception for cluster id: {cluster_id}"
         )
+    except OpinionTypeException:
+        logger.warning(
+            msg=f"Opinion type not found in xml file for cluster id: {cluster_id}"
+        )
 
```

Comment:
I added it because we threw the exception but never showed any message, in the logs that you shared with me I didn't see that exception but I added the message just in case